### PR TITLE
add primary_locale_code and available_primary_locale_codes

### PR DESCRIPTION
**Pull request by bolom**
#4592

added two new attributes
-  App primary language
- available_primary_locale_codes

### CODE REVIEW

**Comment by bolom** [comment]
i have added a link of the issues :(

**Comment by KrauseFx** [comment]
Hey @bolom, thanks for your pull request. It seems like one test is failing. Also, could you add one test for the newly added attributes? 👍

**Comment by bolom** [comment]
@KrauseFx voila :)

File: spaceship/spec/tunes/app_details_spec.rb [hunk 1]
```diff
@@ -35,8 +35,14 @@
       details.secondary_first_sub_category = "Weather"
       expect(details.secondary_first_sub_category).to eq("MZGenre.Weather")
 
-      details.secondary_second_sub_category = "Weather"
-      expect(details.secondary_second_sub_category).to eq("MZGenre.Weather")
+      details.secondary_second_sub_category = "Puzzle"
+      expect(details.secondary_second_sub_category).to eq("MZGenre.Puzzle")
+
+      details.primary_locale_code = "en-US"
```
Comment (KrauseFx) [inline spaceship/spec/tunes/app_details_spec.rb]: Sorry I'm not sure I'm following. Does this test relate to `it prefixes the category with the correct value for all category types`?
Also, just setting the value and retrieving it doesn't seem to actually test the code, instead we should test fetching the value from an existing application, right?

**Comment by bolom** [comment]
@KrauseFx i never use rspec. I have just copy and paste the previous tests

> Sorry I'm not sure I'm following. Does this test relate to it prefixes the category with the correct value for all category types?

not sure what i should test

> Also, just setting the value and retrieving it doesn't seem to actually test the code, instead we should test fetching the value from an existing application, right?

I have though i did that. 

If i can help me, i will do my best to fix the tests

**Comment by bolom** [comment]
@TKBurner any help?

**Comment by KrauseFx** [comment]
Sorry for the late reply, please remove the newly added tests for now, so we can merge this change 👍 
I'll look into tests after that

**Comment by ohwutup** [comment]
Hey @bolom! Thanks for the PR! Just wanted to check in and see if you had a chance to remove those tests and we can merge away! 🚀

**Comment by bolom** [comment]
@ohwutup done

**Comment by ohwutup** [comment]
Hey @bolom, sorry to be so naggy, but it looks like there are still some failing tests in there. Would you mind rebasing on master and pushing up again? That should hopefully fix the broken state. Thanks!

**Comment by bolom** [comment]
@ohwutup all green 👍

**Comment by KrauseFx** [comment]
Looks great, thanks for working on this @bolom 👍

**Comment by bolom** [comment]
@KrauseFx yes ! :)

**Pull request merged**